src/components/filter/Filter.tsx

```diff
@@ -1,130 +1,211 @@
 /* eslint-disable consistent-return */
-import {useHistory, Link} from 'react-router-dom';
-import React, {useState, useEffect} from 'react';
+import React,
+{useState,
+  useEffect}
+  from 'react';
 import './filter.css';
-import {Checkbox} from '@patternfly/react-core/dist/js/components';
-import store from '../redux/store';
+import {Checkbox,
+  Tooltip,
+  Button,
+} from '@patternfly/react-core/dist/js/components';
 import {API_URL} from '../../constants';
+import {InfoCircleIcon} from '@patternfly/react-icons';
+import store from '../redux/store';
+import {FETCH_TASK_SUCCESS} from '../redux/Actions/TaskActionType';
 const Filter: React.FC = (props:any) => {
-  const history = useHistory();
-  const tagsSet = new Set();
-  const [tags, setTags] = useState([]);
-  const [toggle, setToggle] =useState('seeMore');
-  const [i, setI] =useState(10);
+  const [, setTags] = useState();
+  const [status, setStatus]=useState();
+  const [max, setMax]=useState(18);
+  const [, setX]=useState(1); //
+  const [clear, setClear]=useState(' ');
+  const [show, setShow]=useState('seeMore');
+  const tagitem :any =[{id: '1000', value: 'task', isChecked: false},
+    {id: '1001', value: 'pipeline', isChecked: false},
+    {id: '1002', value: 'verified', isChecked: false}];
```

Comment:
why is this hardcoded here?

Reply:
I am getting tags from API only and for other i need to hardcoded.